documentation/design/drafts/output.md

```diff
@@ -0,0 +1,191 @@
+# Github Audit Plugin Output Json Response
+
+Github audit plugin will be providing output in following json format:
+
+## Commits related
+### Type: commit 
+```json
+{
+    "type": "commit",
```

Comment:
Repo information fields are missing. For example, repo_name

Reply:
Added required information